rasa_nlu/model.py

```diff
@@ -141,14 +141,14 @@ def train(self, data):
         context["training_data"] = data
 
         for component in self.pipeline:
-            args = components.fill_args(component.train_args(), context, self.config.as_dict())
+            args = components.fill_args(component.train_args(), context, self.config)
```

Comment:
I'm 100% sure all these blind `self.config.as_dict()` -> `self.config` break a lot of things.